components/GameScreen.vue

```diff
@@ -347,6 +386,11 @@ watch(gameState, async (newState, prevState) => {
       :style="{ maxWidth: gameState?.width + 'px' }"
     >
       <div class="flex flex-row justify-end mb-2 mt-1 mx-4 gap-6">
+        <ButtonLink
+          @click="follow"
+        >
+          Follow Bot
```

Comment:
nit: we use lowercase in other similar links + let's be explicit about which bot it will follow

```suggestion
          follow my bot
```